specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/datamigration.json

```diff
@@ -751,6 +800,163 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{groupName}/providers/Microsoft.DataMigration/services/{serviceName}/serviceTasks/{taskName}": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/subscriptionId"
+        },
+        {
+          "$ref": "#/parameters/groupName"
+        },
+        {
+          "$ref": "#/parameters/serviceName"
+        },
+        {
+          "$ref": "#/parameters/taskName"
+        },
+        {
+          "$ref": "#/parameters/apiVersion"
+        }
+      ],
+      "put": {
+        "summary": "Create or update service task",
+        "description": "The service tasks resource is a nested, proxy-only resource representing work performed by a DMS instance. The PUT method creates a new service task or updates an existing one, although since service tasks have no mutable custom properties, there is little reason to update an existing one.",
+        "tags": [
+          "Service Task resource",
+          "Standard operation",
+          "PUT"
+        ],
+        "operationId": "ServiceTasks_CreateOrUpdate",
+        "x-ms-examples": {
+          "Tasks_CreateOrUpdate": { "$ref": "./examples/ServiceTasks_CreateOrUpdate.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/projectTask"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Service task updated",
+            "schema": {
+              "$ref": "./definitions/Tasks.json#/definitions/ProjectTask"
+            }
+          },
+          "201": {
+            "description": "Service task created",
+            "schema": {
+              "$ref": "./definitions/Tasks.json#/definitions/ProjectTask"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "./definitions/Common.json#/definitions/ApiError"
+            }
+          }
```

Comment:
Is this a long running operation?

Reply:
Yes, same as project task.  They behave the same just that we created a new resource under a different endpoint.

Reply:
These tasks are not tracked resources.
Similar to project task, they are different from the long running operation defined for ARM tracked resource.
(ARM tracked resource long running operations return Azure-AsyncOperation which the client has to query for status - For project and service tasks, we do not follow this.)